examples/logs/plot_hidden_snakes.log

```diff
@@ -0,0 +1,58 @@
+Please be patient. Learning will take 5-20 minutes.
```

Comment:
why did you commit the log files?

Reply:
Those logs are not required. You can discard them. I kept them to document the experiments.